code/game/gamemodes/clock_cult/__clock_defines.dm

```diff
@@ -0,0 +1,15 @@
+var/global/clockwork_construction_value = 0 //The total value of all structures built by the clockwork cult
```

Comment:
These seem kinda superfluous anyway if you keep a list of all clockwork objects. I'd prefer less globals and more counting functions unless these are meant to also count things already deleted. Not a big deal anyway.